.gitignore

```diff
@@ -0,0 +1 @@
+dist/
```

Comment:
The same thing as for todo app. `start` script should be present allowing you to see the changes automatically